helm-chart/dash0-operator/README.md

```diff
@@ -145,7 +145,32 @@ The operator supports the following CRDs
 - `PodMonitor`
 - `ScrapeConfig` with `kubernetesSDConfigs`
 
-The scraping of endpoints requiring authorization is currently not supported, but will be added in an upcoming release.
+#### Authorization
+
+If the scraped endpoints require authorization, it is mandatory to configure mTLS for the communication between the
+OpenTelemetry TargetAllocator and the collectors.
```

Comment:
_question:_ Would it make sense to also explicitly mention the reason for this requirement for mTLS? (I.e. that the target allocator provides the auth credentials for the scraping from the CRs to the collectors in its HTTP response.) It is mentioned briefly in values.yaml, but I think it might be good to add a note about this here as well.

Reply:
added a short note